scripts/paper/generate_stwo_phase44d_source_emission_figure.py

```
@@ -0,0 +1,564 @@
+            raise SystemExit(
+                "rsvg-convert pdf failed for {}: {}".format(
+                    pdf_path, rsvg_pdf.stderr.strip()
+                )
+            )
+
+
+def build_svg(*, rows_by_variant: Dict[str, Dict[str, str]], measured: bool, runs: int) -> str:
+    subtitle = (
+        "Median of {} measured runs. Verification time compares one typed Phase44D source-emission boundary against the lower-layer manifest-plus-compact-proof baseline.".format(
+            runs
+        )
```

Ruff (0.15.11)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)